src/lib.rs

```diff
@@ -3,7 +3,7 @@ pub mod data_structures;
 pub mod algorithms;
 
 #[cfg(test)]
-mod tests {
+mod edit_distance_tests {
```

Comment:
If this works then more descriptive is better. BUT, this should probably be in the file for what its testing.

Reply:
It does work, and is necessary for multiple sub-module "integration" tests that appear to be how Rust / Cargo supports them currently.

https://stackoverflow.com/a/37626704
https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests